pyproject.toml

```diff
@@ -102,6 +106,8 @@ pycryptodome = "~3.10.1"
 pyuwsgi = "~2.0.19.1.post0"
 # YAML parsing and rendering
 PyYAML = "~5.4.1"
+# Python client for Redis database and key-value store (pinned to <4.0.0)
```

Comment:
Better to instead update to `django-redis` 5.x (as in #1115) as that introduces the appropriate pinning of redis-py to <4.0.0, in which case we don't need to have this as an explicit dependency.